### PR TITLE
Moved inspector functionality from EditorNode to InspectorDock.

### PR DESCRIPTION
**Pull request by djrm**

### CODE REVIEW

File: editor/editor_node.h [hunk 1]
```diff
@@ -142,22 +142,10 @@ class EditorNode : public Node {
 		EDIT_REVERT,
 		TOOLS_ORPHAN_RESOURCES,
 		TOOLS_CUSTOM,
-		RESOURCE_NEW,
-		RESOURCE_LOAD,
 		RESOURCE_SAVE,
```
Comment (akien-mga) [inline editor/editor_node.h]: `RESOURCE_SAVE` and `RESOURCE_SAVE_AS` can't be moved together with the rest?

Comment (djrm) [reply editor/editor_node.h]: im not sure, there is more resource saving logic in EditorNode, that wasn't needed to complete the inspector dock rework, i might need to dig deeper into that to see where those functions are needed.

**Comment by reduz** [comment]
Sorry, I did some changes, please fix the conflicts and I will merge

**Comment by djrm** [comment]
@reduz ok, solved.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**